.github/workflows/sphinx-build.yml

```diff
@@ -19,10 +19,35 @@ jobs:
       with:
         python-version: 3.8
 
+    - name: get date for cache
+      id: date
+      run: echo "::set-output name=date::$(date +'%Y-%m-%d')"
+
+    - name: Pip cache
+      uses: actions/cache@v2
+      env:
+        cache-name: pip-documentation
+      with:
+        path: ~/.cache/pip
+        key: py3.8-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
+        restore-keys: |
+          py3.8-${{ env.cache-name }}-${{ hashFiles('**/setup.py') }}
+
+    - name: Sphinx Gallery Cache
+      uses: actions/cache@v2
+      env:
+        cache-name: sphx-glry-documentation
+      with:
+        path: |
+          tutorial/MNIST
+          tutorial/FashionMNIST
```

Comment:
More of a question but could we could limit usages of datasets in the tutorials to just the regular MNIST, not having to additionally download this one?

Reply:
Sure thing. It's so easy to do.